tests/reference/runtime-test_list_01-3ee9b3e.stdout

```diff
@@ -0,0 +1 @@
+The command 'link /NOLOGO /OUT:test_list_01.out test_list_01.out.tmp.o $DIR\src\bin/../runtime\lpython_runtime_static.lib > NUL' failed.
```

Comment:
I think these changes has to be removed!

Reply:
These files were unused anyways I guess. Check https://github.com/lcompilers/lpython/pull/1655/commits/45973741a5ee1a5db7a96525099b7ce4d3769150